teams/tests/test_views.py

```diff
@@ -0,0 +1,71 @@
+from django.test import TestCase, Client
+from default.models import CustomUser
+from hacker.models import HackerInfo
+from teams.models import TeamRoster
+from teams.utility import is_in_team, add_member
+
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+
+from django.http import HttpResponse
+import os
+from dotenv import load_dotenv
+load_dotenv()
+
+
+class TestPublicTeamsDash(TestCase):
+
+    def setUp(self):
+        emails = [
+            'fushiguro@jujutsutech.jp',
+            'aoi@jujutsutech.jp',
+            'yuta@jujutsutech.jp',
+            'yuji@jujutsutech.jp',
+        ]
+        first_names = [
+            'Fushiguro',
+            'Aoi',
+            'Yuta',
+            'Yuji',
+        ]
+
+        team_names = [
+            'tokyo',
+            'kyoto',
+            'special-grade',
+        ]
+
+        for i in range (len(emails)):
```

Comment:
you also could create a hash of objects (email, first_name, team_name) so that you don't have to use a "index" to iterate through. Instead you can loop through the objects themselves. A little cleaner